content/api/e-commerce-solutions/environmental-info/index.html

```diff
@@ -0,0 +1,48 @@
+---
+title: Environmental info
+layout: api
+notanapi: true
+menu:
+  api:
+    parent: e-commerce-solutions
+    identifier: bring-env
+    title: Environmental info
+weight: 2
+---
+
+<section class="dev-docscontent__section">
+  <p class="text-heading">
+    A web component where you can easily present the environmental information
+    from Posten/Bring in your e-commerce. The information can be displayed by adding
+    the script served by us and the custom html element provided below. Optionally a modal
+    window can be triggered where the information will be displayed instead.
```

Comment:
Putting the value even more upfront. Then explanation and options, and then leading into _how_.

```suggestion
    Present carbon offset information from Posten and Bring in your e-commerce solution. 
    A custom web component lets you present the information embedded on a page or in a 
    modal window. This is all done through a script served by us and the custom HTML 
    element as described below.
```

Reply:
Oh, great! Much better! 🙌🏻